code/modules/mob/living/emote.dm

```diff
@@ -263,7 +263,9 @@
 		return
 	else
 		var/sound
-		if(!user.is_muzzled() && !user.mind.miming)
+		var/miming = user.mind ? user.mind.miming : 0

```

Comment:
minor nitpick: but if this issue occurs on other files as well, could you turn this into a define? So we can just use

`is_miming(mob)` in procs to check if the user should act things out instead?